src/typescript/Scala.tmLanguage.ts

```diff
@@ -1153,7 +1156,7 @@ export const scalaTmLanguage: TmLanguage = {
     inheritance: {
       patterns: [
         {
-          match: `\\b(extends|with|derives)\\b\\s*(${idUpper}|${backQuotedId}|(?=\\([^\\)]+=>)|(?=${plainid})|(?="))?`,
+          match: `\\b(extends|with)\\b\\s*(${idUpper}|${backQuotedId}|(?=\\([^\\)]+=>)|(?=${plainid})|(?="))?`,
```

Comment:
```suggestion
          match: `\\b(extends|with|derives|,)\\b\\s*(${idUpper}|${backQuotedId}|(?=\\([^\\)]+=>)|(?=${plainid})|(?="))?`,
```
something like this wouldn't work?

Reply:
I don't think so because it's too generic, that rule accepts any identifiers separated by `,` as `entity.other.inherited-class.scala`.  For example `A , B`, `B` will be `entity.other.inherited-class.scala` though it's not.

It would be great if we can apply such rules only in `extends` and `derives` clause by 

```ts
patterns: [
  {
    start: `\\b(extends|derives)`,
    end: ???,
    patterns: [
      // patterns that matches `qualId`s and assign name as `entity.other.inherited-class.scala`
    ]
  }
]
```

However, the problem is we can't tell what is the `end` of `extends` or `derives`.
(At first, I thought maybe we can end with `:` or `{` or `<newline>` but it doesn't work if derives clause contains new line)
```scala
class A derives B,
  C
```

Reply:
With new line that should be ok since we ended with `,` we still expect something to be after it, no?